README.md

```diff
@@ -6,9 +6,24 @@ Keeping the world on track with _TrainTrack_!
 
 ## Setup
 
-#### Database
+Install mongodb locally (Mac):
 
-Checkout [Mongo DB setup steps](https://github.com/Ledgy/TrainTrack/wiki/Mongo-db-dev-setup).
+```
+brew tap mongodb/brew
+brew install mongodb-community@4.0
+```
+
+To run mongodb once:
+
+```
+mongod --config /usr/local/etc/mongod.conf
+```
+
+To run mongodb started at login:
```

Comment:
Might be better: `To automatically run mongodb after login:` if that's what's meant by this sentence :)

Reply:
good point, addressed in https://github.com/Ledgy/TrainTrack/pull/5/commits/c4aced9fe870b0e2c71850821b887e7eba9fce9a